src/components/grid-smart/grid-smart.tsx

```diff
@@ -58,11 +57,21 @@ export class GridSmart
 
   @Prop() loadingState: "loading" | "loaded";
 
+  /**
+   * Logging level. For troubleshooting component update and initialization.
+   */
+  @Prop() logLevel: "debug" | "off" = "debug";
```

Comment:
I'm not sure if log level should be part of the properties of the component. In any case, I'd rather have a global variable or a static member of a class to enable logging on all the components.

Reply:
Agree!
I'm working on small Logging framework for this in another branch.
So this code will be removed.